### ISSUE TITLE
No BleachBit 3.2.0 rpm package for CentOS 8

### ISSUE THREAD

**Issue by paulqwerty** [bleachbit/bleachbit#791]
The only Bleachbit version listed for CentOS 8 (bleachbit-3.1.0-36.1.noarch.rpm) is not downloadable via the site.

I get a 404 page instead: https://download.bleachbit.org/bleachbit-3.1.0-36.1.centos8.noarch.rpm

However, **the main issue is** that the bleachbit-3.1.0-36.1.noarch.rpm does not work on CentOS 8, I get:

```
Error: 
 Problem: conflicting requests
  - nothing provides python3-scandir needed by bleachbit-3.1.0-36.1.noarch
(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)
```
When trying to install it.

PowerTools and EPEL are installed and enabled.

Thanks

**Comment by az0** [comment]
What about version 3.2.0 from https://www.bleachbit.org/download/linux ?

**Comment by az0** [comment]
That's not a good package, so I took it down. Does the CentOS 7 package  work on CentOS 8? I mean `bleachbit-3.2.0-1.1.centos7.noarch.rpm`

Does CentOS 8 have python2?

**Comment by az0** [comment]
If you like, you can try [BleachBit 3.9.0 for CentOS 8 ](https://build.opensuse.org/package/binary/download/home:andrew_z/bleachbit/CentOS_8/x86_64/bleachbit-3.9.0-1.1.src.rpm). It uses Python 3 instead of Python 3, and the package dependencies are different.

**Comment by az0** [comment]
@paulqwerty 
Are you able to test the BleachBit 3.9.0 package I mentioned a week ago?

**Comment by paulqwerty** [comment]
Hi Andrew,

Yes I did try it and no, I was not able to successfully run it...

The package itself is not installable on Centos 8, so I tried to manually move the files to the (hopefully) correct locations.

But it still did not run...

Hope that helps
Paul

**Comment by az0** [comment]
On the OpenSUSE Build Service (OBS), I remove the scandir preference. I haven't had a chance to test it yet. If you do test it, I would appreciate it. If it doesn't work, please try to give a verbatim error message by copying and pasting or by uploading a screenshot. (GitHub accepts PNG files.)

**Comment by paulqwerty** [comment]
Andrew,

The link you sent earlier, returns a 404...

Do you have a different link?

Cheers
Paul

**Comment by az0** [comment]
1. Click the stable download link for BleachBit packages on [OpenSUSE Build Service](https://software.opensuse.org//download.html?project=home%3Aandrew_z&package=bleachbit). 

1. Click CentOS

1. Click Grab binary packages directly

**Comment by paulqwerty** [comment]
Yep, that worked, thanks.

**Comment by az0** [comment]
Thank you for confirming it worked. By the way, the RPMs are now available through the page [BleachBit 3.9.0 beta](https://www.bleachbit.org/news/bleachbit-390-beta). I now recommend this address over OpenSUSE Build Service that I sent earlier.

**Comment by paulqwerty** [comment]
Andrew,

Confirming, the 3.9.0-5.1 works great!

Thanks a lot!
Paul
